docker/launch_backend_service.sh

```
@@ -118,7 +118,7 @@ run_server(){
     if [[ "${API_PROXY_SCHEME}" == "go" ]]; then
         prepare_for_go
         server_name="ragflow_server"
-        server_cmd=("bin/ragflow_server")
+        server_cmd=("bin/ragflow_server", "--api")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)